modules/core/shared/src/main/scala/lucuma/core/math/Epoch.scala

```diff
@@ -77,13 +78,13 @@ object Epoch extends EpochOptics {
     * Standard epoch.
     * @group Constructors
     */
-  val J2000: Epoch = Julian.fromIntegralYears(2000)
+  val J2000: Epoch = Julian.fromIntegralYears(refineMV[Year](2000))
```

Comment:
When using custom refined types (like `IntYear`), we can define:

`object IntYear extends RefinedTypeOps.Numeric[IntYear, Int]`

which allows us to define instances at compile time as `IntYear(2000)`. At runtime we can use `IntYear.unsafeFrom(xxx): FTP`,  `IntYear.from(t: T): Either[String, FTP]` and `IntYear.unapply(t: T): Option[FTP]`, and a bunch of methods that return `Validated` when we `import eu.timepit.refined.cats.syntax._`.

If the underlying type is not numeric, we can just do `extends RefinedTypeOps[<Refined>, <Underlying>]`.

I don't think this changes anything besides code style, which is of course a matter of preference.